### PR TITLE
All files for both new case study pages

### PR DESCRIPTION
**Pull request by alexcontini**
Sorry I have never done this before and seem to be having a lot of trouble. I forked kubernetes.github.io and then added all the new case study files here. Let me know if there are any problems!

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/4235)

### CODE REVIEW

**Comment by steveperry-53** [comment]
@alexcontini, In the right column, do you see a box labeled Allow edits from maintainers? If so, could you check that box.

**Comment by steveperry-53** [comment]
@alexcontini, You can see that the deploy/netlify check has failed. That often happens because som reference in one of the files can't be found.

**Comment by steveperry-53** [comment]
@alexcontini, I don't think you can use ./ in hrefs.
So change href="./pearson/" to href="/case-studies/pearson/"
and href="./box/" to href="/case-studies/box/"

**Comment by steveperry-53** [comment]
@alexcontini, I don't understand why the jpg files are in this PR. I don't see them referenced anywhere.

**Comment by alexcontini** [comment]
@steveperry-53 my extreme apologies for all this confusion, it's my first time doing this and I promise to learn quickly and not repeat mistakes next time. 

- I updated the index.html file with the ./ replaced by /case_studies/

- I checked "allow edits by maintainers"

- the jpeg files are background images for sections. Reference to them is found in the css files (style_box.css and style_golfnow.css)

- I added 2 new png files (logos for each page)

**Comment by steveperry-53** [comment]
On [this page](https://deploy-preview-4235--kubernetes-io-vnext-staging.netlify.com/case-studies/), the GolfNow image covers up some text. And the Read about GolfNow link goes to the Pearson case study.

[This page](https://deploy-preview-4235--kubernetes-io-vnext-staging.netlify.com/case-studies/golfnow/) has some rendering issues. For example, in the right column, pieces of text cover each other. And at the upper left, the Kubernetes box is smashed up against some headings.

[This page](https://deploy-preview-4235--kubernetes-io-vnext-staging.netlify.com/case-studies/box) has the same rendering issues as golf-now. Also, check the text that starts with, "The vast majority." Is the background for that text supposed to be the Box UI?

**Comment by alexcontini** [comment]
@steveperry-53 thank you very much! so I updated the box.html css file (style_box.css), going to see if the updates work and I will update the other css file as well. 

 - So this is how box.html is supposed to look: https://alexcontini.github.io/box.html - I changed the width of the quotes (ie. banner2text) as they are not going far enough, and that jumbled text on the top right side is not part of the file (I think it says Blog and Get Started, etc.)
- The logo up top should be on the right site of "Case Study:", not sure why it is looking like that? The logo files are both in "images/" and it is linked to "/images/box_logo.png" so not sure why it's not showing up? 
- The background of the whole page should be white, I added body { background-color:#ffffff; in the css file as well to turn that dark grey pattern into white instead.
- I also reduced the width of the golfnow.png logo that appears on the landing page so it should not cut off the text like that

Thank you so much for all your help!

**Comment by chenopis** [comment]
@alexcontini Looks like the problem is that if you use `layout: basic`, it wraps your html in a template that adds a header, footer, and additional CSS. Goto the preview page https://deploy-preview-4235--kubernetes-io-vnext-staging.netlify.com/case-studies/box/ and view the source. I will try to create a more stripped down layout for you to use.

File: case-studies/box.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+      <link href="https://fonts.googleapis.com/css?family=Roboto:300,400" rel="stylesheet">
+      <link rel="stylesheet" type="text/css" href="/css/style_box.css">
+    <title>Box Case Study</title>
+  </head>
+<body>
+
+
+<div class="banner1">
+  <h1>CASE STUDY: <img src="box_logo.png" class="logo" width="10%" style="margin-bottom:-6px"><br>
```
Comment (chenopis) [inline case-studies/box.html]: I think your image path should be `src="/images/box_logo.png"`.

File: case-studies/golfnow.html [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+      <link href="https://fonts.googleapis.com/css?family=Roboto:300,400" rel="stylesheet">
+      <link rel="stylesheet" type="text/css" href="/css/style_golfnow.css">
+    <title>GolfNow Case Study</title>
+  </head>
+<body>
+
+  <div class="banner1">
+    <h1>CASE STUDY: <img src="golfnow_logo.png" class="logo" width="20%" style="margin-bottom:-6px"><br>
```
Comment (chenopis) [inline case-studies/golfnow.html]: Image path should be: `src="/images/golfnow_logo.png"`

File: case-studies/box.html [hunk 3]
```diff
@@ -0,0 +1,127 @@
+---
+title: Box Case Study
+layout: basic
```
Comment (chenopis) [inline case-studies/box.html]: Try changing this to: `layout: none`

File: case-studies/golfnow.html [hunk 4]
```diff
@@ -0,0 +1,138 @@
+---
+title: GolfNow Case Study
+layout: basic
```
Comment (chenopis) [inline case-studies/golfnow.html]: Try changing this to: `layout: none`

**Comment by chenopis** [comment]
@alexcontini Ok, I've created a layout that shouldn't add anything else to your HTML. This is just a temporary fix because I know we need to get this out the door. Once it's up, I can go back and figure out how to make a better layout to suit your content.

**Comment by chenopis** [comment]
Ok, all the changes are live now:
* https://kubernetes.io/case-studies/
* https://kubernetes.io/case-studies/golfnow/
* https://kubernetes.io/case-studies/box/

@alexcontini Email me \@google.com so we can coordinate a more long-term solution for this.

**Comment by alexcontini** [comment]
@chenopis, thank you so much for all your help, they both look perfect. I am going to email you now to figure out the layout issue. Thanks again to everyone for getting these up

**Pull request merged**